### ISSUE TITLE
AttributeError: XYZ object has no attribute '_class__'

### ISSUE THREAD

**Issue by jzelez** [opbeat/opbeat_python#26]
I think you have an error in Django Middleware on line 62. 
It should say `view_name = self.thread_local.view_func.__class__.__name__`. 
Currently it's missing one underscore in front of class.

It errors out with my configuration which includes newrelic:
```
Traceback (most recent call last):
  File ".../local/lib/python2.7/site-packages/opbeat/contrib/django/middleware/__init__.py", line 81, in process_response
    self.thread_local.view_func)
  File ".../local/lib/python2.7/site-packages/opbeat/contrib/django/middleware/__init__.py", line 62, in _get_name_from_view_func
    view_name = self.thread_local.view_func._class__.__name__
  File ".../local/lib/python2.7/site-packages/newrelic-2.44.0.36/newrelic/common/object_wrapper.py", line 50, in __getattr__
    return _ObjectProxy.__getattr__(self, name)
AttributeError: XYZ object has no attribute '_class__'
```

**Comment by roncohen** [comment]
Well that's embarrassing. I'll fix it right away

**Comment by jzelez** [comment]
Thanks ;)

**Comment by roncohen** [comment]
@jzelez Thanks for reporting this. It's fixed in 2.0.1. Give it a try!

**Comment by jzelez** [comment]
Yup. All cool now. ;)
